### DISCUSSION TITLE
0 Class "ComPagesEventSubscriberDispatcher" not found

### DISCUSSION THREAD

**Discussion by desbear** [joomlatools/joomlatools-pages#967] [Q&A] [answered]
after I update to com_pages_v0.23.1. Joomla show this error message
 0 Class "ComPagesEventSubscriberDispatcher" not found
![CleanShot-2022-09-01-at-16-46-45](https://user-images.githubusercontent.com/545068/187873484-63565681-608e-42a3-a46c-5bd7cbba8cf1.jpg)

**Comment by johanjanssens** [comment, marked as answer]
@desbear I haven't done much testing in Joomla for 0.23, focussed mostly on getting it to work standalone. What version of Joomla are you using?

**Reply by desbear** [reply]
Joomla! 3.10.11  php 7.4.21

**Reply by johanjanssens** [reply]
Thank, this is likely a refactoring left-over. See: https://github.com/joomlatools/joomlatools-pages/blob/f33744046c57c6dafd4d548814474cc5f9cd2c05/contrib/extensions/joomla/event/subscriber/dispatcher.php#L10 

Replace `ComPagesEventSubscriberDispatcher` with `ComPagesEventSubscriberAbstract`, which should fix it.

**Reply by johanjanssens** [reply]
Pages 0.23.2 is now released which includes a fix for this.
